app/helpers/application_helper.rb

```diff
@@ -145,4 +145,26 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def event_types(conference)
+    all = conference.program.event_types.map { |et| et.title.pluralize }
+    first = all[0...-1]
+    last = all[-1]
+    ets = ''
+    if all.length > 1
+      ets << first.join(', ')
+      ets << " and #{last}"
+    else
+      ets = all.join
+    end
+    ets
+  end
```

Comment:
We can do this but this results in
`['one', 'two', 'three'].to_sentence # => "one, two, and three"` (if we need the comma)
or a better way 
`['one', 'two', 'three'].to_sentence(last_word_connector: ' and ') # => "one, two and three"`
http://apidock.com/rails/Array/to_sentence